src/lib.rs

```diff
@@ -160,7 +160,7 @@ pub use self::dma::DMA;
 pub use self::gpio::GPIO;
 #[cfg(feature = "82x")]
 pub use self::i2c::I2C;
-pub use self::mrtimer::MRTimer;
+pub use self::mrtimer::MRT;
```

Comment:
Module is still called `mrtimer`, which is inconsistent with its contents. Please rename to `mrt`.

Reply:
Should™ be fine now, after a careful check with `grep`